static/js/alert_words_ui.js

```diff
@@ -111,3 +114,7 @@ export function set_up_alert_words() {
         alert.hide();
     });
 }
+
+export function reset() {
+    loaded = false;
```

Comment:
This function is not actually called anywhere yet. And the `loaded` value is also not used. See other settings files like `settings_emoji.js` for how we use the `loaded` variable.

Reply:
can you please look at this once again, i think i've called this `reset` function in `node_tests` for `alert-words`.

Reply:
which is how we're calling this `reset` function in all of our other `.js` files for settings.

Reply:
The reset is not called in the original code. And it can be used in tests but the current tests does not make sense as it only checks value of `loaded` variable. We can use `reset` function to test that the `render_alert_words_ui` function returns early when loaded is false which we do not do currently.

Also we call `reset` function in `settings_sections.reset_sections` as well for organization settings.

Reply:
hmm, i missed this `settings_sections.reset_sections` part, thanks :) , have updated the PR with it.